test/unit/test-batched-json.js

```diff
@@ -134,26 +132,37 @@ describe('batchFetchJsonFor', () => {
           });
     });
 
-    it('should send POST request with crossorigin attribute present with no' +
-        ' identity token', () => {
+    it('should send POST request with empty, defined identity token', () => {
+      const el = element('https://data.com');
+      const all = UrlReplacementPolicy.ALL;
+
+      urlReplacements.expandUrlAsync
+          .withArgs('https://data.com')
+          .returns(Promise.resolve('https://data.com'));
+
+      return batchFetchJsonFor(ampdoc, el, null, all, false, '')
+          .then(() => {
+            const fetchArgs = fetchJson['firstCall']['args'];
+            expect(fetchArgs[1]['body']['ampViewerAuthToken'])
+                .to.equal('');
+            expect(fetchArgs[1]['method'])
+                .to.equal('POST');
```

Comment:
Nit: Prefer dot syntax, or use the same "expectedRequest" format in other tests.

```js
const fetchOpt = fetchJson.firstCall.args[1];
expect(fetchOpt.body.ampViewerAuthToken).to.equal('');
expect(fetchOpt.method).to.equal('POST');
```